packages/shared/src/components/badges/TopReaderBadge.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { ReactElement } from 'react';
+import { DevCardTheme, themeToLinearGradient } from '../profile/devcard';
+import LogoIcon from '../../svg/LogoIcon';
+import LogoText from '../../svg/LogoText';
+import type { LoggedUser } from '../../lib/user';
+import type { Keyword } from '../../graphql/keywords';
+import {
+  Typography,
+  TypographyColor,
+  TypographyType,
+} from '../typography/Typography';
+import { BadgeIcon } from './BadgeIcon';
+
+export const TopReaderBadge = ({
+  user,
+  issuedAt,
+  keyword,
+}: {
+  user: Pick<LoggedUser, 'name' | 'image' | 'username'>;
+  issuedAt: Date;
+  keyword: Pick<Keyword, 'value' | 'flags'>;
+}): ReactElement => {
+  const { name, username, image } = user;
+  const formattedDate = new Date(issuedAt).toLocaleString('en-US', {
+    year: 'numeric',
+    month: 'long',
+  });
+
+  return (
+    <div
+      className="flex h-80 w-80 rounded-24 p-1 text-center text-text-primary"
+      style={{
+        backgroundImage: themeToLinearGradient[DevCardTheme.Gold],
+      }}
+    >
+      <div className="flex max-w-full flex-1 flex-col items-center rounded-20 bg-background-default bg-[url('https://daily-now-res.cloudinary.com/image/upload/s--hQ68GsbF--/f_auto,q_auto/v1730356819/webapp/image_7')] bg-cover px-3 py-5">
```

Comment:
Can we put this image inside `cloudinary` export?
Just makes it easier to have it all in one place really.

Reply:
I can't use variables inside tailwind custom values, I could add it as a custom tailwind class for this specifically.
Or should I just use `style={{}}`?

Reply:
Changed to use img